doc/api/n-api.md

```diff
@@ -1552,6 +1584,36 @@ This API returns various properties of a typed array.
 *Warning*: Use caution while using this API since the underlying data buffer
 is managed by the VM
 
+
+
+#### *napi_get_dataview_info*
+<!-- YAML
+added: v8.0.0
+-->
+```C
+napi_status napi_get_dataview_info(napi_env env,
+                                     napi_value dataview,
+                                     size_t* bytelength,
+                                     void** data,
+                                     napi_value* arraybuffer,
+                                     size_t* byte_offset)
+```
+
+- `[in] env`: The environment that the API is invoked under.
+- `[in] dataview`: `napi_value` representing the DataView whose
+properties to query.
+- `[out] length`: Number of elements in the DataView.
+- `[out] data`: The data buffer underlying the DataView.
+- `[out] byte_offset`: The byte offset within the data buffer from which
+to start projecting the DataView.
+
+Returns `napi_ok` if the API succeeded.
+
+This API returns various properties of a DataView.
+
+*Warning*: Use caution while using this API since the underlying data buffer
+is managed by the VM
```

Comment:
I don’t really know what this is saying? If you do use the `napi_value* arraybuffer` parameter, this should be okay, right?